hw1/02-palindrome.html

```diff
@@ -6,20 +6,26 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body style="background-color: blue">
```

Comment:
You can add the `bg-primary` class to use Bootstrap's default blue color:
https://getbootstrap.com/docs/5.0/utilities/background/#background-color

Reply:
I didn't know about this, worked great. thanks!